pkg/detectors/sendbird/sendbird_test.go

```diff
@@ -79,14 +82,65 @@ func TestSendbird_FromChunk(t *testing.T) {
 				data:   []byte("You cannot find the secret within"),
 				verify: true,
 			},
-			want:    nil,
-			wantErr: false,
+			want:                nil,
+			wantErr:             false,
+			wantVerificationErr: false,
+		},
+		{
+			name: "found, would be verified if not for timeout",
+			s:    Scanner{client: common.SaneHttpClientTimeOut(1 * time.Microsecond)},
+			args: args{
+				ctx:    context.Background(),
+				data:   []byte(fmt.Sprintf("You can find a sendbird application ID %s with sendbird secret %s within", appId, secret)),
+				verify: true,
+			},
+			want: []detectors.Result{
+				{
+					DetectorType: detectorspb.DetectorType_Sendbird,
+					Verified:     false,
+				},
+			},
+			wantErr:             false,
+			wantVerificationErr: true,
+		},
+		{
+			name: "found, verified but unexpected api surface",
+			s:    Scanner{client: common.ConstantResponseHttpClient(404, "")},
```

Comment:
There's a comment in the verification code that indicates that Sendbird returns a 400 for all errors. Does that include what we'd normally expect a 404 for? If so, this test should probably test a 500 or 502 or 503 or something (and be renamed).

Reply:
no, we can still get a 404. for example if we called an endpoint that didn't exist (maybe bc the api changed) we can get a 404. in that case, we just don't get an extra error code in the response (which we mostly get with 400, but also a few other status codes: 403, 429, 500, 503)